cmake/Modules/FindLibRealSense.cmake

```diff
@@ -0,0 +1,64 @@
+###############################################################################
+# Find Intel librealsense
+#
+#     find_package(LIBREALSENSE)
+#
+# Variables defined by this module:
+#
+#  LIBREALSENSE_FOUND                 True if librealsense was found
+#  LIBREALSENSE_INCLUDE_DIRS          The location(s) of librealsense headers
+#  LIBREALSENSE_LIBRARIES             Libraries needed to use librealsense
+
+find_package(PkgConfig QUIET)
+
+# Include directories
+find_path(LIBREALSENSE_INCLUDE_DIR librealsense2/rs.h
+          PATHS "$ENV{realsense2_DIR}"
+                "$ENV{PROGRAMW6432}/librealsense2" # for self built library 
+                "$ENV{PROGRAMFILES}/librealsense2" # for self built library
+                "$ENV{PROGRAMFILES}/Intel RealSense SDK 2.0" # for pre built library
+                # "Please specify search paths for Linux and MacOS"
+          PATH_SUFFIXES include)
+
+set(LIBREALSENSE_INCLUDE_DIRS ${LIBREALSENSE_INCLUDE_DIR})
+mark_as_advanced(LIBREALSENSE_INCLUDE_DIRS)
+
+# Libraries
+set(LIBREALSENSE_RELEASE_NAME realsense2)
+set(LIBREALSENSE_DEBUG_NAME realsense2_d)
```

Comment:
Of course, you are right about the name of the libraries (librealsense.lib vs realsense2) 👍 

I also liked your recommended update to include PATHS and PATH_SUFFIXES in the calls to find_path and find_library, so I've updated that as well. I also don't know what appropriate values would be on linux/mac -- hopefully a Linux/Mac user can comment with additional recommendations for those platforms